src/NServiceBus.NHibernate.Tests/API/APIApprovals.cs

```diff
@@ -1,18 +1,43 @@
 ﻿namespace NServiceBus.NHibernate.Tests.API
 {
+    using System;
+    using System.Linq;
     using System.Runtime.CompilerServices;
     using NUnit.Framework;
     using PublicApiGenerator;
 
     [TestFixture]
     public class APIApprovals
     {
+#if NET461
         [Test]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public void Approve()
+        public void Approve__NET461()
         {
-            var publicApi = ApiGenerator.GeneratePublicApi(typeof(NHibernatePersistence).Assembly);
+            var publicApi = Filter(ApiGenerator.GeneratePublicApi(typeof(NHibernatePersistence).Assembly));
             TestApprover.Verify(publicApi);
         }
+#endif
+
+#if NETCOREAPP2_0
```

Comment:
Are the approvals different on each platform?

If it's just the TargetFrameworkAttribute getting you down, you can follow this recipe, and then the file is the same for both platforms.
https://github.com/Particular/NServiceBus.Persistence.Sql/blob/develop/src/SqlPersistence.Tests/APIApprovals.cs